scratch2/blocks.js

```diff
@@ -103,6 +103,7 @@ IconView.prototype.draw = function() {
 
 IconView.icons = {
   greenFlag: { width: 20, height: 21, dy: -2 },
+  stopSign: { width: 20, height: 20, dx: 2, dy: 0 },
```

Comment:
Does `dx` actually do anything here? I'm not sure I can tell a difference!

This seems fine to me:
```suggestion
  stopSign: { width: 20, height: 20 },
```